modules/sqs/main.tf

```diff
@@ -3,7 +3,7 @@ locals {
   tags = merge(var.tags, { Deployment = var.prefix })
 }
 
-## SQS IAM access policy for internal-report-queue SQS
+## SQS IAM access policy for internal-report-queue.fifo SQS
```

Comment:
Per our discussion on FIFO and how a FIF) DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.